src/index.html

```diff
@@ -29,6 +29,10 @@ <h1 data-locale="bookmarks">Bookmarks</h1>
     </nav>
   </section>
 
+  <section id="feedback" draggable="false">
+    <a draggable="false" href="https://github.com/ear1grey/structured-start-tab/issues">Feedback</a>
+  </section>
+
```

Comment:
Feedback button/link should be:
* Switchable on/off from the options.
* Also permanently available in options.
* Internationalised (as should all the sidebar buttons).